free_fleet_client_ros1/src/ClientNode.cpp

```diff
@@ -283,6 +304,23 @@ bool ClientNode::read_mode_request()
       paused = false;
       emergency = true;
     }
+    else if (mode_request.mode.mode == messages::RobotMode::MODE_CHARGING)
```

Comment:
Tested with a Kobuki robot by adding a `MODE_DOCKING`. This feature is similarly helpful to trigger a docking sequence with the same `TriggerSrv`.

Reply:
so would you recommend just adding an `or` check? Or does it have it's own service to be used?

Reply:
https://github.com/open-rmf/free_fleet/pull/74/commits/901e39f806bbf4165938db87d345196cdb3552eb